cadence/lib/js/src/kibble.js

```diff
@@ -0,0 +1,143 @@
+import { Address, UFix64 } from "@onflow/types";
+import {
+	deployContractByName,
+	executeScript,
+	getContractAddress,
+	getScriptCode,
+	getTransactionCode,
+	mintFlow,
+	sendTransaction,
+} from "flow-js-testing";
+
+import { getRegistry } from "./common";
+
+/*
+ * Deploys FungibleToken and Kibble contracts to Registry.
+ * @throws Will throw an error if transaction is reverted.
+ * @returns {Promise<*>}
+ * */
+export const deployKibble = async () => {
+	const Registry = await getRegistry();
```

Comment:
Very minor, but it wasn't immediately obvious to me that `Registry` is an account

Reply:
Replaced it with `KittyAdmin` :)